README.md

```diff
@@ -19,7 +19,13 @@ answer, including your reasoning, to this markdown file.
 
 
 ### Answer
-The implementation has a worst-case complexity of O(V+E), where V is the number of nodes and E is the number of edges. 
+The implementation has a worst-case complexity of O(V+E), where V is the number of vertices and E is the number of edges. 
 
 
-The overall time complexity is O(V+E), with constant time complexities of O(1) for Graph constructor and addEdge methods, O(E) for isCyclicUtil function, and O(V) for isCyclic method.
+1. Graph constructor and addEdge are time complexity of constant time count as O(1)
```

Comment:
What data structure you used for this? Adjacency List or Matrix?

Reply:
Adjacency List

Reply:
O(1) if the ajacency list adding an edge